commands/curation/curationaudit_test.go

```diff
@@ -529,24 +541,25 @@ func getTestCasesForDoCurationAudit() []testCase {
 			requestToFail: map[string]bool{
 				"/api/go/go-virtual/rsc.io/sampler/@v/v1.3.0.zip": false,
 			},
-			expectedResp: map[string][]*PackageStatus{
-				"github.com/you/hello": {{
-					Action:            "blocked",
-					ParentName:        "rsc.io/quote",
-					ParentVersion:     "v1.5.2",
-					BlockedPackageUrl: "/api/go/go-virtual/rsc.io/sampler/@v/v1.3.0.zip",
-					PackageName:       "rsc.io/sampler",
-					PackageVersion:    "v1.3.0",
-					BlockingReason:    "Policy violations",
-					DepRelation:       "indirect",
-					PkgType:           "go",
-					Policy: []Policy{
-						{
-							Policy:    "pol1",
-							Condition: "cond1",
+			expectedResp: map[string]*CurationReport{
+				"github.com/you/hello": &CurationReport{packagesStatus: []*PackageStatus{
```

Comment:
```suggestion
				"github.com/you/hello": {packagesStatus: []*PackageStatus{
```
redundant type from array, slice, or map composite literal (for all in this file)